src/Switch.js

```diff
@@ -0,0 +1,3 @@
+export function Switch(props, children) {
+  return children[0]
```

Comment:
How does this work 🤔

Reply:
I was wondering the same thing. Shouldn't it be the first **matched** child?

Reply:
Oh, wait, `<Route />` only renders if the path matches, so I think the `children` array would really have only 1 item.

Reply:
@lukejacksonn @vdsabev Yes, exactly. Switch ensures at least one element is rendered and if no route matches, then `children[0]` will be undefined, so it works.